packages/schematics/angular/utility/project-targets.ts

```diff
@@ -6,15 +6,31 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import { experimental } from '@angular-devkit/core';
+import { SchematicsException, Tree } from '@angular-devkit/schematics';
+import { getProject, isWorkspaceProject } from './project';
+import { WorkspaceProject, WorkspaceSchema, WorkspaceTarget } from './workspace-models';
 
+export function getProjectTargets(project: WorkspaceProject): WorkspaceTarget;
```

Comment:
Could we do without these two first signatures? The third one seems to be a superset of them which makes me think it'd be the only one needed.